awscli/customizations/s3/utils.py

```diff
@@ -353,6 +353,44 @@ def relative_path(filename, start=os.path.curdir):
         return os.path.abspath(filename)
 
 
+def is_glacier_object(response_data):
```

Comment:
Is there a reason this needs to be extracted out?  Wouldn't it make more sense in fileinfo?